index.bs

```diff
@@ -5338,18 +5348,24 @@ The <dfn id="dfn-typed-array-type" export>typed array types</dfn> are
 {{Uint16Array}},
 {{Uint32Array}},
 {{Uint8ClampedArray}},
-{{Float32Array}} and
-{{Float64Array}}.
+{{Float32Array}},
+{{Float64Array}},
+any [=nullable types=] whose [=nullable types/inner type=] is a [=typed array type=],
+and any [=annotated types=] whose [=annotated types/inner type=] is a [=typed array type=].
 
-The <dfn id="dfn-buffer-source-type" export>buffer source types</dfn>
-are {{ArrayBuffer}},
+The <dfn id="dfn-buffer-source-type" export>buffer source types</dfn> are
```

Comment:
With this part changed, the claim that "buffer source types are simply references to objects" stops being true, no?

And "Values of the IDL buffer source types are represented by objects of the corresponding ECMAScript class" also stops beign true.